### ISSUE TITLE
MacOS settings to play MIDI

### ISSUE THREAD

**Issue by djaychela** [geokala/music_tutorial#8]
With a freshly-downloaded and installed VLC (3.0.8 in my case), the following allows MIDI output to work:

**environment.set('midiPath', '/Applications/VLC.app/Contents/MacOS/VLC')**

**Comment by geokala** [comment]
I've updated the master branch with some code to deal with this. Can you give it a try if you get a chance?
I'm still a bit iffy about the midi_path being for a specific version of timidity, but I'm not sure if there's something better we can do with it?

**Comment by geokala** [comment]
Oh, whoops, I see now that you put this to use VLC, I'll update it to work with VLC instead now.

**Comment by geokala** [comment]
Right, it should use VLC+MuseScore on OSX now, and give helpful output if it can't use those.

**Comment by djaychela** [comment]
Just downloaded it into a fresh venv and it worked straight away using VLC and musescore.  I'll try it on another mac with neither installed when I can next week and see how it reacts to their absence.

**Comment by geokala** [comment]
Fantastic.

Without either app installed it should give you helpful error messages, hopefully!
